examples/stable_diffusion_v2/ldm/util.py

```diff
@@ -55,3 +57,8 @@ def extract_into_tensor(a, t, x_shape):
     b = t.shape[0]
     out = ops.GatherD()(a, -1, t)
     return out.reshape(b, *((1,) * (len(x_shape) - 1)))
+
+def is_old_ms_version(last_old_version="1.10.1"):
```

Comment:
mindnlp implements as from _legacy import ops/nn, then nn.Dropout, maybe we follow in later version

Reply:
that's another option that requires to wrap all uncompatible ops.